piglow/piglow.go

```diff
@@ -34,29 +30,24 @@ func (p *PiGlow) Enable() error {
 	return p.conn.Write([]byte{0x00, 0x01})
 }
 
-// Setup enables normal operations, resets the internal registers, and enables
+// setup enables normal operations, resets the internal registers, and enables
 // all LED control registers
-func (p *PiGlow) Setup() error {
```

Comment:
if `Setup` stays exported (see #26) I would suggest to rename it. To me this is almost the real `Reset`. I don't know what the 3 registers do (doc would have been nice) but it sounds like `Setup` would be called the first time around or every time we want a clean slate, it does sound like a reset.